build

```diff
@@ -12,29 +12,23 @@ if [ -z ${VERSION+a} ]; then
 	VERSION=$(git describe --dirty --always)
 fi
 
-GLDFLAGS+="-X github.com/coreos/ignition/internal/version.Raw=${VERSION}"
-
-if [ ! -h gopath/src/${REPO_PATH} ]; then
-	mkdir -p gopath/src/${ORG_PATH}
-	ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
-fi
+GLDFLAGS+="-X github.com/flatcar-linux/ignition/internal/v0/version.Raw=${VERSION}"
```

Comment:
Why `internal/v0/version` and not `v0/internal/version` ? :thinking:

Reply:
Ahh good catch, indeed the version is not correctly set:
```
./bin/amd64/ignition --version
Ignition was not built properly
```

it's better with the right order:
```
./bin/amd64/ignition --version
Ignition v0.34.0-45-gf7f5e498-dirty
```

:coffee: